tests/unit/Actions.spec.ts

```diff
@@ -102,7 +82,9 @@ describe('Action button states', () => {
   })
 
   it('renders buttons once Entity Type is known', async () => {
-    await setEntityType('BEN')
+    store.setEntityType(CorpTypeCd.BENEFIT_COMPANY)
+
+    await Vue.nextTick()
```

Comment:
Did you add this because the test failed otherwise, or did you add it because we were previously awaiting the mutation?

Any idea if we always have to await actions now?

Reply:
In this example, it is returning a promise, since it makes a network call and doesn't resolve until that's done.

So, do we need to await some actions?

And should we type actions differently when we import them? cc: @jonathan-longe

Reply:
https://pinia.vuejs.org/core-concepts/actions.html

Reply:
The Pinia page doesn't have an example of using their async action. I assume it needs to be awaited.